guide/17-geocoding-guides/part7_utility_functions_for_geocode.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>The utility function <code>analyze_geocode_input()</code> analyzes a file and auto-generates a field mapping for the input table or file provided, and then outputs GeocodeParameters <code>JSON</code>, which is a required parameter for the <code>batch_geocode</code> tool and can be used to investigate/improve/fine tune <code>batch_geocode</code> operation. </p><p> </p><p>For other ways to perform analyze actions, please check:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>